app/helpers/sessions_helper.rb

```diff
@@ -65,7 +65,8 @@ def authenticate_session_user user_id
     user = User.find_by id: user_id
     return unless user
 
-    if session[:session_token] && user.authenticated?(session[:session_token])
+    if session[:session_token] && user.authenticated?(:remember,
+                                                      session[:session_token])
       user
```

Comment:
```suggestion
    user if user.authenticated?(:remember, session[:session_token])
```
Viết gọn lại được, bên trong authenticated có check nil rồi